README.md

```diff
@@ -1,2 +1,45 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+int MAX_GUESSES = 4
+string TO_GUESS = "Word to guess"
+bool won = False
+print("Guess the " + TO_GUESS.length() + " letter word")
+print("_" * TO_GUESS)
+for(int y = 0; y < MAX_GUESSES; y++)
```

Comment:
Personally I'm not a big fan of C-style for loops in pseudocode -- I prefer something more readable like
```
for y = 0, 1, ..., MAX_GUESSES-1
```
To an extent this is a matter of personal preference -- just worth bearing in mind that you don't have to adhere too closely to real programming language syntax when writing pseudocode